components/financial_assistance/app/views/financial_assistance/incomes/_other_income_form.html.erb

```diff
@@ -1,54 +1,60 @@
-  <%= render partial: 'financial_assistance/shared/temporary_warning_message', locals: { form_object_id: income_id, translation_key: "faa.income_temporary_message", translation_interpolation_keys: {current_year: TimeKeeper.date_of_record.year.to_s, next_year: (TimeKeeper.date_of_record + 1.year).year.to_s} } %>
-<% end %>
-<div class="focus_effect_module">
-  <div class="row row-form-wrapper no-buffer row-height row-edit-border">
-    <%= f.hidden_field :kind, value: kind %>
-    <% if (other_income.kind == "social_security_benefit" || kind == :social_security_benefit) && FinancialAssistanceRegistry.feature_enabled?(:ssi_income_types) %>
-      <div class="row row-form-wrapper no-buffer check_box_collection" id="ssi_type">
-        <div class="col-xs-12 form-group form-group-lg ss_benefit_type">
-           <%= f.label :frequency_kind, "#{l10n("Social").upcase} #{l10n("Security").upcase} #{l10n("Benefit").upcase} #{l10n("TYPE").upcase} *", class: "label-color" %>
-           <%= f.select(:ssi_type, options_for_select(FinancialAssistance::Income::SS_BENEFITS.map {|k, v| [v, k]}, selected: other_income.ssi_type), {prompt: "Choose"}, {disabled: false, required: true}) %>
+
+  <% onchange_date = FinancialAssistanceRegistry[:income_and_deduction_date_warning].enabled? ? (raw "validateDateWarnings('#{income_id}')") : "" %>
+  <div class="focus_effect_module">
+    <div class="row row-form-wrapper no-buffer row-height row-edit-border">
+      <%= f.hidden_field :kind, value: kind %>
+      <% if (other_income.kind == "social_security_benefit" || kind == :social_security_benefit) && FinancialAssistanceRegistry.feature_enabled?(:ssi_income_types) %>
+        <div class="row row-form-wrapper no-buffer check_box_collection" id="ssi_type">
+          <div class="col-xs-12 form-group form-group-lg ss_benefit_type">
+            <%= f.label :frequency_kind, "#{l10n("Social").upcase} #{l10n("Security").upcase} #{l10n("Benefit").upcase} #{l10n("TYPE").upcase} *", class: "label-color" %>
+            <%= f.select(:ssi_type, options_for_select(FinancialAssistance::Income::SS_BENEFITS.map {|k, v| [v, k]}, selected: other_income.ssi_type), {prompt: "Choose"}, {disabled: false, required: true}) %>
+          </div>
+        </div>
+      <% end %>
+      <div class="col-md-3">
+        <%= f.label :frequency_kind, 'Amount *', class: "amount label-color" %>
+        <div class="faa-amount">
+          <%= f.text_field :amount, placeholder: '0.00', value: number_to_currency(other_income.amount), required: true %>
         </div>
       </div>
-    <% end %>
-    <div class="col-md-3">
-      <%= f.label :frequency_kind, 'Amount *', class: "amount label-color" %>
-      <div class="faa-amount">
-        <%= f.text_field :amount, placeholder: '0.00', value: number_to_currency(other_income.amount), required: true %>
+      <div class="col-md-3 fa-select select-box">
+        <%#= f.select_tag :frequency, placeholder: 'HOW OFTEN *', required: true, class: "floatlabel" %>
```

Comment:
if you end up back in this file, delete this

Reply:
👍